docs/source/index.rst

```diff
@@ -0,0 +1,84 @@
+.. Cleanlab documentation master file, created by
+   sphinx-quickstart on Mon Jan 10 07:17:00 2022.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Introduction
+============
+
+**Cleanlab automatically finds and fixes errors in your ML datasets.**
+
+| This reduces manual work needed to fix data issues and helps train reliable ML models on partially mislabeled datasets. Cleanlab has already found thousands of `label errors <https://labelerrors.com>`_ in ImageNet, MNIST, and other popular ML benchmarking datasets, so let's get started with yours!
+
+.. note::
+   Cleanlab currently only supports classification tasks.
+
+
+Quickstart
+==========
+
+1. Install Cleanlab.
+--------------------
+
+.. tabs::
+
+   .. code-tab:: py pip
+      
+      pip install cleanlab
+
+   .. code-tab:: py conda
+
+      conda install -c conda-forge cleanlab
+
+2. Find label errors with ``get_noise_indices``.
+------------------------------------------------
+
+Cleanlab's ``get_noise_indices`` function tells you which examples in your dataset are likely mislabeled. At a minimum, it expects two inputs - your data's labels, ``y``, and its out-of-sample predicted probabilities, ``pyx``, computed with cross-validation. 
```

Comment:
This statement is unclear:  
its out-of-sample predicted probabilities, ``pyx``, computed with cross-validation

I'd replace it with:
predicted probabilities, ``pyx``, from some trained model (Note: these must be out-of-sample predictions where the datapoints were held-out from the model during training, which can be obtained via cross-validation).


""

Reply:
your data's labels
-> 
your data's given labels

Reply:
done!